spec/stat_tracker_spec.rb

```diff
@@ -38,8 +38,17 @@
    describe '#read_game_teams_csv' do
       it 'create game_team objects' do
          location = './data/game_teams_fixture.csv'
+         game_team_0 = @stat_tracker.read_game_teams_csv(location)[0]
```

Comment:
Same thing here. Maybe test the last one as well so we can have a more robust testing. But overall, this is great!